.env.example

```diff
@@ -22,8 +22,8 @@ MAIL_USERNAME=null
 MAIL_PASSWORD=null
 MAIL_ENCRYPTION=null
 
-NORTHSTAR_URI=http://northstar-qa.dosomething.org
+NORTHSTAR_URI=http://northstar.dev:8000
 NORTHSTAR_API_KEY=abc4324
 
-PHOENIX_URI=https://staging.beta.dosomething.org/api
+PHOENIX_URI=https://dev.dosomething.org:8888
 PHOENIX_VERSION=v1
```

Comment:
Maybe change this key to be `PHOENIX_API_VERSION` to match the northstar key and to indicate it's for the API :)